libraries/AP_Baro/AP_Baro_BMP085.cpp

```diff
@@ -33,11 +33,33 @@ extern const AP_HAL::HAL &hal;
 #define OVERSAMPLING BMP085_OVERSAMPLING_HIGHRES
 #endif
 
-AP_Baro_BMP085::AP_Baro_BMP085(AP_Baro &baro, AP_HAL::OwnPtr<AP_HAL::I2CDevice> dev)
+AP_Baro_BMP085::AP_Baro_BMP085(AP_Baro &baro, AP_HAL::OwnPtr<AP_HAL::Device> dev)
```

Comment:
Do we have any variant of BMP* that can be in a SPI bus rather than I2C?

Reply:
just copied from some anothe driver